src/main/java/com/google/firebase/messaging/AndroidNotification.java

```diff
@@ -309,6 +414,185 @@ public Builder setImage(String imageUrl) {
       return this;
     }
 
+    /**
+     * Sets the "ticker" text, which is sent to accessibility services. Prior to API level 21
+     * (Lollipop), sets the text that is displayed in the status bar when the notification 
+     * first arrives.
+     *
+     * @param ticker Ticker name.
+     * @return This builder.
+     */
+    public Builder setTicker(String ticker) {
+      this.ticker = ticker;
+      return this;
+    }
+
+    /**
+     * Sets the sticky flag. When set to false or unset, the notification is automatically 
+     * dismissed when the user clicks it in the panel. When set to true, the notification 
+     * persists even when the user clicks it.
+     *
+     * @param sticky The sticky flag
+     * @return This builder.
+     */
+    public Builder setSticky(boolean sticky) {
+      this.sticky = sticky;
+      return this;
+    }
+
+    /**
+     * Sets the time that the event in the notification occurred in milliseconds. Notifications
+     * in the panel are sorted by this time. The time will be formated in RFC3339 UTC "Zulu" 
+     * format, accurate to nanoseconds. Example: "2014-10-02T15:01:23.045123456Z". Note that 
+     * since the time is in milliseconds, the last section of the time representation always
+     * has 6 leading zeros.
+     *
+     * @param eventTimeInMillis The event time in milliseconds
+     * @return This builder.
+     */
+    public Builder setEventTimeInMillis(long eventTimeInMillis) {
+      this.eventTime = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSSSSSSS'Z'")
+          .format(new Date(eventTimeInMillis));
+      return this;
+    }
+
+    /**
+     * Sets whether or not this notification is relevant only to the current device. Some 
+     * notifications can be bridged to other devices for remote display, such as a Wear 
+     * OS watch. This hint can be set to recommend this notification not be bridged.
```

Comment:
Is it really just a recommendation?

Reply:
The behavior of this flag on the client side is documented at https://developer.android.com/reference/android/app/Notification.Builder#setLocalOnly(boolean) But on FCM API side since we can't control the client OS version, we don't know what the actual behavior will be on the client side. 
But I agree the word "recommend" is misleading here. Maybe just add a link here to https://developer.android.com/reference/android/app/Notification.Builder#setLocalOnly(boolean) ? (Or if adding a link is hard, just remove the last sentence?